python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -661,7 +664,14 @@ def get_best_logdir(
                 based on `mode`, and compare trials based on `mode=[min,max]`.
         """
         best_trial = self.get_best_trial(metric, mode, scope)
-        return best_trial.logdir if best_trial else None
+        if best_trial and best_trial.relative_logdir:
```

Comment:
For backwards compatibility

```suggestion
        if best_trial and hasattr(best_trial, "relative_logdir") and best_trial.relative_logdir:
```

Reply:
I'd suggest for brevity:
```
if best_trial and getattr(best_trial, "relative_logdir", None):
```

Reply:
Updated

Reply:
Done